pipelines/fixtures/resource-types-with-params.yml

```diff
@@ -0,0 +1,93 @@
+---
+resource_types:
+- name: custom-resource-type
+  type: git
+  privileged: ((privileged))
+  source:
+    uri: {{origin-git-server}}
+    branch: master
+  params:
+    foo: bar
```

Comment:
Is this actually observed/verified in the tests somehow? In other words, would these tests fail without the implementation, or would this just be silently ignored?

This may be difficult to verify, to be honest; the [git resource params](https://github.com/concourse/git-resource#parameters) only has a few params that we could really detect being used, and even then, this only applies to the resource type fetching...

Maybe we could do something wacky like point it to a repo containing some of the resource type's code as a submodule? But it even defaults to fetching them by default, so we could only really turn it off and see that it fails when the params are respected. Weird stuff.